android/assets/jsons/translations/template.properties

```diff
@@ -990,6 +990,13 @@ Found Pantheon =
 Follow [belief] = 
 Religions and Beliefs = 
          
+# Religion overview screen
+Religion Name: = 
+Founding Civ: = 
+??? = ???
```

Comment:
Love it. Tempts to slip in a nice translation for that somewhere... `??? = ❓❓❓` would probably show just fine.